app/assets/javascripts/application.js

```diff
@@ -27,3 +27,4 @@
 // For blacklight_range_limit built-in JS, if you don't want it you don't need
 // this:
 //= require 'blacklight_range_limit'
+//= requiere 'focus'
```

Comment:
Shouldn't this be `//= require`?

Reply:
Yes, absolutely. I'm actually surprised it still works. Is it necessary?